frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -77,36 +85,50 @@ const getColumns = () => {
   ];
 };
 
+const isNodeUnschedulable = (node: NodeKind) => {
+  const taints = getNodeTaints(node);
+  const taintsList = [NO_SCHEDULE_EFFECT, PREFER_NO_SCHEDULE_EFFECT, NO_EXECUTE_EFFECT];
+
+  return taints.some((taint: string) => taintsList.includes(_.get(taint, 'effect')));
```

Comment:
Isn't `taintsList` the full lists of possible effects? Couldn't this just be...?

```suggestion
  return !_.isEmpty(node.spec.taints);
```